guiEditor/src/guiNodeTools.ts

```diff
@@ -18,7 +18,7 @@ import { RadioButton } from "babylonjs-gui/2D/controls/radioButton";
 import { ImageBasedSlider } from "babylonjs-gui/2D/controls/sliders/imageBasedSlider";
 
 export class GUINodeTools {
-    public static ImageControlDefaultUrl = "https://playground.babylonjs.com/textures/grass.png";
+    public static ImageControlDefaultUrl = "https://playground.babylonjs.com/textures/imageControlDefault.jpg";
```

Comment:
maybe the playground is not the best place for this. This should be in the assets library instead

Reply:
awesome :-)